test/dependencies/DependenciesItem.test.tsx

```diff
@@ -10,11 +10,18 @@ describe("<DependenciesItem />", () => {
   beforeEach(() => {
     component = render(
       mockTheme(
-        <DependenciesItem
-          mode="edit"
-          dependency={DEPENDENCY}
-          handleClick={handleClick}
-        />
+        <table>
```

Comment:
Why do we need to wrap this in a table element etc now?

Reply:
Because the component outputs a `<td>`. The framework will print a warning about how `<td>` cannot be a direct child of a `<div>`. It's not strictly necessary to wrap this component in a table element, but it suppresses the warning and also helps make it clear that this component should be used as part of a table, not standalone.